src/abi/stack.h

```diff
@@ -68,9 +65,13 @@ getStackSpace(Index local, Function* func, Index size, Module& wasm) {
     WASM_UNREACHABLE("unhandled pointerType");
   }
   block->list.push_back(builder.makeGlobalSet(stackPointer->name, added));
+  block->list.push_back(builder.makeLocalSet(local, added));
```

Comment:
This is the cause of the error on CI, I think - the `added` node is reused here, but each node is only allowed to appear once in the IR.

The sub operation could be stored in another local but maybe there is a nicer way.

Reply:
> This is the cause of the error on CI, I think - the `added` node is reused here, but each node is only allowed to appear once in the IR.

i didn't know that. thank you. fixed.

> The sub operation could be stored in another local but maybe there is a nicer way.

i guess it can be something like:
```
global.get $sp
i32.const xxx
i32.sub
local.tee yyy
global.set $sp
```
but i'm not sure how it can be done with binaryen IR builder API.

Reply:
You can use `builder.makeLocalTee` to replace the `makeLocalSet/Get` pair here. That is, something like `makeGlobalSet(makeLocalTee(..))`.

Reply:
thank you. done.